source/_integrations/homekit.markdown

```diff
@@ -543,6 +543,24 @@ You can also try to use `avahi-daemon` in reflector mode together with the optio
 
 Configure the network mode as `networkbridge`. Otherwise the Home Assistant Bridge won't be exposed to the network.
 
+#### Accessory does not appear in the Home App (for pairing) - Libvirt QEMU/KVM virtual machine with macvtap adapter
+
```

Comment:
Since this affects all multicast/zeroconf traffic, does it make sense to move this into the `zeroconf` docs (https://www.home-assistant.io/integrations/zeroconf/) and then link to it here?

Reply:
That indeed sounds like a better and more modular approach. I'll update my PR in a bit to reflect this. Are there any other blocks in the HomeKit documentation which would be better in the zeroconf docs?